preview-src/index.ts

```diff
@@ -161,7 +160,10 @@ if (settings.scrollEditorWithPreview) {
       scrollDisabled = false
     } else {
       const line = getEditorLineNumberForPageOffset(window.scrollY)
-      if (typeof line === 'number' && !isNaN(line)) {
+      if (window.scrollY === 0) {
```

Comment:
You should fetch origin and rebase on `origin/master`, this change does not belong to this pull request as it's already on master: 

https://github.com/asciidoctor/asciidoctor-vscode/blob/da84b8af7366696aa26a0c16951fa13501b45a04/preview-src/index.ts#L163-L165

Reply:
See below, I've had difficulty rebasing this and would prefer to either squash and rebase or open another PR.